src/lib/url.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Validate the url
+ * @param value The Url String
+ * @returns The parsed boolean
+ */
+export function validateUrl(value?: string | number | null): boolean {
+  try {
+    if (!value || typeof value !== "string") {
+      return false;
+    }
+
+    if (!value.startsWith("http://") && !value.startsWith("https://")) {
```

Comment:
Limit to only consider "http/https" protocols might not be good for a `validateUrl` method, maybe renaming the method to `isValidHttpUrl` (this naming also matches the other validation methods like isObject, isValidGuid,...) might be better.

And at this point it might be good to have both, `validateUrl` and `validateHttpUrl`, and then one can reuse the other and do additional checks, something like:
 
```
export function isValidUrl(value?: string): boolean {
  try {
    new URL(value);
    return true;
  } catch {
    return false;
  }
}

export function isValidHttpUrl(value?: string): boolean {
  if (!isValidUrl(value)) {
    return false;
  }

  return value.startsWith("http");
}
```

what you think?